packages/contracts/src/core/dao/DAO.sol

```diff
@@ -127,6 +127,7 @@ contract DAO is
     }
 
     /// @notice Disables the initializers on the implementation contract to prevent it from being left uninitialized.
+    /// @custom:oz-upgrades-unsafe-allow constructor
```

Comment:
These surprisingly became necessary for OZ's `upgrades` package to not throw errors in the added tests.

Reply:
don't we need to clearly explain why we are silencing/disabling this warning/error

Reply:
all good here